src/main.rs

```diff
@@ -504,8 +514,13 @@ async fn make_app(current_dir: &std::ffi::OsString) -> anyhow::Result<Command> {
             )
         )
         .subcommand(Command::new("dev-ui")
-            .about("Start the web UI development server with hot reloading (same as `cd ui && npm i && npm start`)")
+            .about("Start the web UI development server with hot reloading (same as `cd ui && npm i && npm run dev`). To instead create a production build, use the --release flag.")
             .visible_alias("d")
+            .arg(Arg::new("RELEASE")
+                .action(ArgAction::SetTrue)
+                .long("release")
+                .help("If set, create a production build")
+            )
```

Comment:
This should probably come after at least some of the other args since this affects the print order of the helptext.

Reply:
[f3d1590](https://github.com/kinode-dao/kit/pull/112/commits/f3d15901d642319c59e048b2d0671f3170653801)